### PR TITLE
[Bi 22] Create UI Login page

### PR DESCRIPTION
**Pull request by Lucian2**
image

### CODE REVIEW

File: src/api/config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let API_HOST = 'https://api-internship.binarcode.com';
```
Comment (PascuMihai) [inline src/api/config.js:1]: not sure why this file is here

File: package.json [hunk 2]
```diff
@@ -8,10 +8,14 @@
     "lint": "vue-cli-service lint"
```
Comment (PascuMihai) [inline package.json:8]: Not sure why this file is still here, did't you have these changes before?

File: src/views/Login.vue [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    <div class="sm:mx-auto sm:w-full sm:max-w-md">
+      <img class="mx-auto h-12 w-auto" src="https://tailwindui.com/img/logos/workflow-mark-indigo-600.svg" alt="Workflow" />
+      <h2 class="mt-6 text-center text-3xl font-extrabold text-gray-800">
+         {{ $t('register.button.label.signIn') }}
+      </h2>
+    </div>
+    <div class="mt-8 sm:mx-auto sm:w-full sm:max-w-md">
+      <div class="bg-white py-8 px-4 shadow sm:rounded-lg sm:px-10">
+           <ValidationObserver v-slot="{ valid }">
+        <form class="space-y-6" action="#" method="POST" @submit.prevent="handleSubmit">
```
Comment (PascuMihai) [inline src/views/Login.vue:12]: you can remove action and type of method. is not necessary

File: src/views/Login.vue [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+          <div>
+            <div class="mt-1">
```
Comment (PascuMihai) [inline src/views/Login.vue:14]: Why do you have the nested divs here, they can be removed, and style the input or add a wrapper div on every input if you want to set the container width

File: src/views/Login.vue [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+              :name="$t('general.name.email')"
+              :label="$t('general.name.email')"
+              rules="required|email"
+              :placeholder="$t('general.placeholder.email')"
+            />
+            </div>
+          </div>
+          <div>
+            <div class="mt-1">
+
```
Comment (PascuMihai) [inline src/views/Login.vue:28]: same here, no need for the nested divs

File: src/views/Login.vue [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+              :name="$t('general.name.password')"
+              :label="$t('general.name.password')"
+              rules="required|min:6"
+              :placeholder="$t('password')"
+            />
+            </div>
+          </div>
+          <div class="flex items-center justify-between">
+            <div class="text-sm">
+              <a href="#" class="font-medium text-indigo-600 hover:text-indigo-500">
```
Comment (PascuMihai) [inline src/views/Login.vue:41]: No need of all these nested divs, and also replace the a tag with router link

**Review by Ceciliaa — APPROVED** [review]

**Review by PascuMihai — APPROVED** [review]

**Pull request merged**